### PR TITLE
reopening PR for Microsoft CAS

### PR DESCRIPTION
**Pull request by ChanochShayner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/19868

## Description
Integration to Microsoft cloud app security.
continuation of - https://github.com/demisto/content/pull/8047

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by ChanochShayner** [comment]
adding docs approved as it was added from original PR
FYI @ShirleyDenkberg

**Comment by guykeller** [comment]
Please fix conf.json before merging

File: Packs/MicrosoftCloudAppSecurity/Classifiers/classifier-MicrosoftCloudAppSecurity_5_9_9.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+	"id": "575cb861-e842-4e01-8379-1c9f55bd3b38",
```
Comment (ronykoz) [inline Packs/MicrosoftCloudAppSecurity/Classifiers/classifier-MicrosoftCloudAppSecurity_5_9_9.json:2]: update the id to be `MicrosoftCAS`

File: Packs/MicrosoftCloudAppSecurity/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "MicrosoftCloudAppSecurity",
+    "description": "MicrosoftCloudAppSecurity",
```
Comment (ronykoz) [inline Packs/MicrosoftCloudAppSecurity/pack_metadata.json:3]: update the description to be more informative

File: Packs/MicrosoftCloudAppSecurity/TestPlaybooks/MicrosoftCloudAppSecurity-Test.yml [hunk 3]
```diff
@@ -0,0 +1,224 @@
+id: MicrosoftCloudAppSecurity-Test
+version: -1
+vcShouldKeepItemLegacyProdMachine: false
```
Comment (ronykoz) [inline Packs/MicrosoftCloudAppSecurity/TestPlaybooks/MicrosoftCloudAppSecurity-Test.yml:3]: remove this line

File: Packs/CommonTypes/ReleaseNotes/1_9_0.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#### Incident Fields
+##### Username
+##### Policy Type
+##### Risk Score
+##### Cloud Service
+##### Account Name
+##### Policy Description
+##### Country
+add Microsoft CAS Alert to this incident fields
```
Comment (ronykoz) [inline Packs/CommonTypes/ReleaseNotes/1_9_0.md:9]: this doesn't seem like the right format

File: Packs/MicrosoftCloudAppSecurity/Classifiers/classifier-MicrosoftCloudAppSecurity_5_9_9.json (hunk 1 above)
Comment (guykeller) [reply Packs/MicrosoftCloudAppSecurity/Classifiers/classifier-MicrosoftCloudAppSecurity_5_9_9.json:2]: Actually, let's fix the IDs everywhere.
If the `format` command doesn't do that, let's open a dev bug

**Pull request merged**